featuretools/tests/entityset_tests/test_ww_es.py

```diff
@@ -79,6 +77,13 @@ def test_init_es_with_woodwork_table(df):
     assert es['table'].ww.logical_types['category'] == Categorical
 
 
+def test_init_es_with_woodwork_table_diff_name_error(df):
+    df.ww.init(index='id', name='table')
+    error = "Cannot add dataframe with conflicting dictionary key, diff_name, from dataframe name, table"
```

Comment:
how about "Naming conflict in dataframes dictionary:  dictionary key 'diff_name' does not match dataframe name 'table'"?

Reply:
updated